tests/sm/networkmanager/networkmanager_test.cpp

```diff
@@ -80,13 +76,13 @@ class NetworkManagerTest : public Test {
         return std::string(std::istreambuf_iterator<char>(file), std::istreambuf_iterator<char>());
     }
 
-    StorageMock                                  mStorage;
-    std::unique_ptr<CNIMock>                     mCNI;
-    TrafficMonitorMock                           mTrafficMonitor;
-    NetworkManager                               mNetManager;
-    std::unique_ptr<NamespaceManagerMock>        mNetns;
-    std::unique_ptr<NetworkInterfaceManagerMock> mNetIf;
-    aos::StaticString<aos::cFilePathLen>         mWorkingDir;
+    StorageMock                          mStorage;
+    CNIMock                              mCNI;
+    TrafficMonitorMock                   mTrafficMonitor;
+    NetworkManager                       mNetManager;
+    NamespaceManagerMock                 mNetns;
```

Comment:
why you delete StrictMock ? this good to monitor if all mock called as expected, in case if we forgot to call mock test will be failed, and I did it to be sure that all mock function will be called in case of error add instance to network